#global testerror "Err0303"

func toto()
{
    let a = 1._5    
}